projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/plugin_entry.py

```diff
@@ -27,9 +37,22 @@ def run_job(context: JobContext) -> None:
     @staticmethod
     @hookimpl
     def vdk_configure(config_builder: ConfigurationBuilder) -> None:
+        """
+        Adds the DAG-related configuration variables.
+
+        :param config_builder: the builder used to add the configuration variables
+        :return:
+        """
         add_definitions(config_builder)
 
 
 @hookimpl
 def vdk_start(plugin_registry: IPluginRegistry, command_line_args: List):
+    """
+    Loads the DAGs plugin.
+
+    :param plugin_registry: plugin registry used to register the plugin and its hooks
+    :param command_line_args: a list of command line args passed to VDK
+    :return:
+    """
     plugin_registry.load_plugin_with_hooks_impl(MetaJobsPlugin(), "MetaJobsPlugin")
```

Comment:
nit: My recommendation is, not to spend redundant effort in documenting trivial code blocks, so that it's less noisy. In case anything in particular has to be said, then providing documentation is excellent.

Reply:
I agree I might have overkilled it slightly. Should I remove the already written kinda obvious docs or take it as a recommendation for similar future efforts?

Reply:
Removed some obvious docs.